### DISCUSSION TITLE
Notifications?

### DISCUSSION THREAD

**Discussion by mondalaci** [HandyGram/app#11] [Q&A]
I'm running HandyGram 04.2, but it doesn't provide messages on my SM-R965 Galaxy Watch 6 Classic 47mm LTE.

Should HandyGram make Telegram notifications appear on my watch?

**Comment by slazze** [comment]
Notifications are not supported in older versions. You need to download “Alpha 1”.

**Reply by tonirigatoni7-jpg** [reply]
I downloaded the alpha and I still don't get notifications on my Galaxy Watch 7. I have notifications enabled, but I still don't get notifications. Is there something else I am missing? The watch is only Bluetooth/Wifi compatible and I don't know if certain connections need to be enabled. I have tried every combination of having my connections enabled, but still no luck.
